### PR TITLE
changed readme and renamed the job

### PR DESCRIPTION
**Pull request by smootherbug**

### CODE REVIEW

**Review by kishorv10 — APPROVED** [review]

**Comment by MadhuriArugula** [comment]
@taz-pega-work can you please review and approve this PR

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This project provides Helm charts and basic examples for deploying Pega on Kubernetes. You will also need to download the required [installation kit](https://community.pega.com/knowledgebase/products/platform/deploy) from the Pega Community which includes rules and data to preload into your relational database. Deploying Pega on Kubernetes requires Pega Infinity 8.2 or newer.
```
Comment (taz-pega-work) [inline README.md:3]: can you also fix this terminology so it's consistent with our version descriptions?
Pega Infinity 8.2 or newer
to Pega Infinity 8.2 and later.

**Review by taz-pega-work — APPROVED** [review]
minor change requested but I approve

File: README.md (hunk 1 above)
Comment (smootherbug) [reply README.md]: updated

**Pull request merged**